Activities/Directory.build.targets

```diff
@@ -1,23 +1,30 @@
 <Project>
-  <ItemGroup Condition="$(TargetFramework.StartsWith('net5'))"> 
-    <PackageReference Update="System.Activities" Version="5.0.0-20210615-01" />
-    <PackageReference Update="UiPath.Workflow" Version="5.0.0-20210615-01" />
-    <PackageReference Update="System.Activities.Core.Presentation" Version="5.0.0-20210724.1" />
-    <PackageReference Update="System.Activities.ViewModels" Version="1.0.0-20220307.1"/>
-  </ItemGroup>
-  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.1'"> 
-    <PackageReference Update="System.Activities" Version="1.0.0-20201105-02" />
-    <PackageReference Update="UiPath.Workflow" Version="1.0.0-20201105-02" />
-    <PackageReference Update="System.Activities.Core.Presentation" Version="1.0.0-alpha003" />
-  </ItemGroup>
-  <ItemGroup>
-	<PackageReference Update="UiPath.Robot.Api" Version="22.4.1" PrivateAssets="All" />
-	<PackageReference Update="UiPath.Activities.Api" Version="22.4.1" PrivateAssets="All" />
-	<PackageReference Update="UiPath.Activities.Contracts" Version="22.4.1" PrivateAssets="All" />
-	<PackageReference Update="UiPath.Telemetry.Client" Version="1.3.18" PrivateAssets="All" />
-	<PackageReference Update="UiPath.Platform" Version="22.4.1"  PrivateAssets="All" />
-  </ItemGroup>
-  <PropertyGroup>
-    <DefineConstants Condition="'$(TargetFramework)' == 'net5.0'">$(DefineConstants);NETSTANDARD</DefineConstants>
-  </PropertyGroup>
+	<ItemGroup Condition="$(TargetFramework.StartsWith('net6'))">
+		<PackageReference Update="System.Activities" Version="6.0.0-20220106-04" />
```

Comment:
Update this to `UiPath.Workflow.Runtime` and match versions as in this document: https://docs.uipath.com/developer/docs/migrating-activities-to-net-5#:~:text=%3CPackageReference%20Update%3D%22UiPath.Workflow.Runtime%22%20Version%3D%226.0.0%2D20220401%2D03%22%20PrivateAssets%3D%22All%22/%3E